Library/Formula/icmake.rb

```diff
@@ -0,0 +1,42 @@
+class Icmake < Formula
+  desc "Make utility using a C-line grammar"
+  homepage "https://fbb-git.github.io/icmake/"
+  url "https://github.com/fbb-git/icmake/archive/7.23.02.tar.gz"
+  sha256 "f030086d0630c2c1755519556b9cf88b8495bc8debf36b257e0c69d3b199c46b"
+
+  depends_on "gnu-sed" => :build
```

Comment:
This might be a run time dependency, too. Parts of `icmake` shell out to `sed` when you run it, and they may use GNU-specific behavior. When fiddling with 7.22, I had to [repoint it at `gsed`](https://github.com/apjanke/homebrew-zsh/blob/ff79b989ec00717b8aa53cf21d94efd7a9d34ab3/icmake.rb#L24-L27) to get it to work for building `yodl`.